schematic/models/validate_attribute.py

```diff
@@ -31,6 +31,49 @@
 ScopeTypes = Literal["set", "value"]
 
 
+@dataclass
+class ValueValidationOutput:
+    """
+    This is the result of doing cross validation when of type value.
+
+    Args:
```

Comment:
nit: Technically these are `Attributes:`, rather than `Args:`

Reply:
For dataclasses they are both right?

Reply:
https://google.github.io/styleguide/pyguide.html#384-classes shows `Attributes`, and that's what I've consistently used.

Reply:
Done!